### PR TITLE
Redirecting view for buyer

### PR DESCRIPTION
**Pull request by sarias-eb**

### CODE REVIEW

File: users_app/views.py [hunk 1]
```diff
@@ -43,8 +43,10 @@ class LoginView(TemplateView):
     def get(self, request, *args, **kwargs):
         user = self.request.user
         if user.is_authenticated():
-            if user.is_AP:
+            if user.is_AP :
                 return HttpResponseRedirect(reverse('ap-taxpayers'))
+            elif user.is_buyer:
+                return HttpResponseRedirect(reverse('company-list'))
```
Comment (nicolasnunez-eb) [inline users_app/views.py:49]: I think we should add a new test for this new redirection

Comment (sarias-eb) [reply users_app/views.py:49]: Thanks!

File: users_app/tests/test_user.py [hunk 2]
```diff
@@ -123,6 +124,13 @@ def test_login_success_with_Google_should_redirect_to_apsite(self):
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertEqual('supplier_app/ap-taxpayers.html', response.template_name[0])
 
+    def test_login_success_with_Google_should_redirect_to_buyersite(self):
```
Comment (ricardocaif-eb) [inline users_app/tests/test_user.py:127]: Good job! Don't know if it belongs to this PR but we should consider making test cases for the views that this new user is not allowed to get in.

Comment (sarias-eb) [reply users_app/tests/test_user.py]: This new user uses urls that already have tests

**Pull request closed without merging**